ref_ts/ts_utils.py

```diff
@@ -359,7 +361,7 @@ def map_waterml(xml_doc):
         values = ts['values']
     elif not version:
         return False
-
+#called by view: create_ref_time_series
```

Comment:
Place method documentation within the docstring. See [PEP8](https://www.python.org/dev/peps/pep-0008/#documentation-strings).